src/backend/_scss/application.scss

```diff
@@ -1672,7 +1672,7 @@ a {
 
   input {
     line-height: 30px;
-    width: 100%;
+    width: 98%;
```

Comment:
This feels like a workaround. Please tweak the paddings, but this and above width should be 100%

Tomorrow, something else might change layout and someone might have to change the width to 96 or 95%

It is not a sustainable solution. Markup must be fixed

Reply:
if we could make the position of the div  absolute and change according to that ,also taking care of the responsiveness 

is this the right approch?